internal/process/provisioning/create_runtime_resource_step.go

```diff
@@ -341,6 +346,45 @@ func (s *CreateRuntimeResourceStep) getEmptyOrExistingRuntimeResource(name, name
 	return &runtime, nil
 }
 
+func (s *CreateRuntimeResourceStep) createKubernetesConfiguration(operation internal.Operation) imv1.Kubernetes {
+	oidc := gardener.OIDCConfig{
+		ClientID:       &s.oidcDefaultValues.ClientID,
+		GroupsClaim:    &s.oidcDefaultValues.GroupsClaim,
+		IssuerURL:      &s.oidcDefaultValues.IssuerURL,
+		SigningAlgs:    s.oidcDefaultValues.SigningAlgs,
+		UsernameClaim:  &s.oidcDefaultValues.UsernameClaim,
+		UsernamePrefix: &s.oidcDefaultValues.UsernamePrefix,
+	}
+	if operation.ProvisioningParameters.Parameters.OIDC != nil {
+		if operation.ProvisioningParameters.Parameters.OIDC.ClientID != "" {
```

Comment:
Could you use `DefaultIfParamNotSet`?

Reply:
unfortunately no, it expects param as a pointer

Reply:
I got it.